docs/content/en/docs/references/api/grpc.md

```diff
@@ -19,11 +19,11 @@ We also generate the [reference doc for the HTTP layer]({{<relref "/docs/referen
 
 
 
-<a name="skaffold.proto"></a>
```

Comment:
The comments at the top of this file references `proto/markdown.tmpl`.  Maybe `hack/generate_proto.sh` needs to be fixed up?